lib/route53.js

```diff
@@ -7,6 +7,52 @@ const Promise = require("bluebird");
 const CustomError = require("./errors")
 const DNSHandler = require("./base")
 
+
+function changeRequestMethod(params, method) {
+	params.ChangeBatch.Changes[0].Action = method;
+	return params;
+}
+
+function generateParams(hostedZoneId, domain, opts) {
+	let params = {
+		HostedZoneId: hostedZoneId,
+		ChangeBatch: {
+			Changes: [
+				{
+					Action: "",
+					ResourceRecordSet: {
+						Name: opts.name.concat(".", domain, "."),
+						Type: opts.type || "CNAME",
+						TTL: opts.ttl || 3600,
+					},
+				},
+			],
+		},
+	};
+
+	if (opts.values) {
+		params.ChangeBatch.Changes[0].ResourceRecordSet.ResourceRecords = opts.values.map(function(r){
```

Comment:
if you wanted to be more succinct, you could use a fat arrow function here. Something like:
```javascript
 = opts.values.map( r => { return {Value: r}; });
```

Really just a matter of opinion there.

Reply:
Just as a note, it is hard to make this particular case more succinct since you are trying to return an object. I think the parser might be confuse by the single line return:
`map( r => {Value:r} );`
You can give it a shot though if you want.